array/js/array.js

```diff
@@ -0,0 +1,88 @@
+function isArray(object) {
+  return object.constructor.name == "Array";
+}
+
+function isArray2(object) {
+  return {}.toString.call(object) == "[object Array]";
```

Comment:
are you sure it works?
`SyntaxError: expected expression, got '.' `is in my console